charts/consul/templates/crd-proxydefaults.yaml

```diff
@@ -143,6 +143,14 @@ spec:
                       type: object
                     type: array
                 type: object
+              failoverPolicy: 
+                description: FailoverPolicy specifies the exact mechanism used for failover.
+                properties:
+                  mode:
+                    description: Mode specifies the type of failover that will be performed. 
+                      Valid values are "default", "" (equivalent to "default") and "order-by-locality".
```

Comment:
I'm currently changing the name of the "default" case to "sequential" in this PR https://github.com/hashicorp/consul-enterprise/pull/4734/files#diff-dadef19b7fe31571431416cb4db25634e4f83cad2b9547e543fa0277e8b4f221R1437

Reply:
I just remembered that we are adding the regions field in that PR too

Reply:
Thanks, updated the value, and will add the extra field when api package is updated.

Reply:
No worries. Thanks!